tests/unit-tests/test-class-utils.php

```diff
@@ -519,4 +519,77 @@ public function testGetTargetResumeId_WhenCalled_ReturnsCourseIdIfItHasNoLessons
 		/* Assert */
 		$this->assertEquals( $course_id, $result );
 	}
+
+	public function testIsFseThemeSupported_WhenCalled_WorksIfBlockTemplatesIndexAvailable() {
```

Comment:
```suggestion
	public function testIsFseTheme_WhenBlockTemplatesIndexAvailable_ReturnsTrue() {
```

The explanation of the suggestion below.